### DISCUSSION TITLE
Feature request: Generate multiple bolt12 offers

### DISCUSSION THREAD

**Discussion by Dolu89** [ACINQ/phoenixd#81] [Ideas]
As initially requested in this issue #80 it could make sense to allow generating multiple bolt12 offers from the same phoenixd instance.

**Comment by pm47** [comment]
This is definitely doable. The reason why we didn't implement it is that those offers would not be deterministically generated from seed and need to be stored in database. So if you move to a different server and restore only from seed, phoenix will not be able to accept payments to these offers. Funds would not be lost, but payments would be rejected. The default offer is deterministic, does not need to be stored in db and will survive a restore from seed.

**Reply by Dolu89** [reply]
Ok I see. It's a user problem to properly backup these data, but I understand phoenixd abstract all the complexity of managing a lightning node.
You could warn that the offers no longer work if a user restore only from seed. It's a risk I can accept as user.
